wagtail/admin/templates/wagtailadmin/pages/editor_sidebar_panels/includes/action_list_item.html

```diff
@@ -0,0 +1,29 @@
+{% load wagtailadmin_tags %}
+<section class="w-py-6" aria-labelledby="status-sidebar-{{ title|cautious_slugify }}">
+
+    {% block content %}
+        <div class="w-flex w-space-x-3">
+            {% icon name=icon_name class_name='w-w-5 w-h-5 w-text-primary w-flex-shrink-0' %}
+
+            <div class="w-flex w-flex-1 w-items-start w-justify-between">
+                <div class="w-flex w-flex-col w-flex-1">
+                    <h3 id="status-sidebar-{{ title|cautious_slugify }}" class="w-label-1 !w-mt-0 w-mb-1">
+                        <div class="w-sr-only">{{ screen_reader_title_prefix }}</div>
+                        {{ title }}
+                    </h3>
+                    {% if help_text %}<p class="w-help-text w-inline-flex w-mb-0">{{ help_text }}</p>{% endif %}
+                </div>
+                {% block action %}
+                    <a class="w-text-14 w-text-teal-200 w-no-underline hover:w-underline w-transition {% if hide_action %}w-hidden{% endif %}"
```

Comment:
Could we not have no `<a>` tag at all if `hide_action`?

```suggestion
{% if hide_action %}
                    <a class="w-text-14 w-text-teal-200 w-no-underline hover:w-underline w-transition"
```